niimpy/EDA/EDA_categorical.py

```diff
@@ -26,6 +26,7 @@ def get_xticks_(df):
     text : list of strings
         xtick text for plotting
     """
+    assert isinstance(df, pd.Series), "df is not a pandas series."
```

Comment:
I guess the name of the variable `df` might be misleading as it implies that the type of `df` is `pd.DataFrame` rather than `pd.Series`. I think `s` or `sr` or `ser`  might be a better choice. Take a look at [this part](https://pandas.pydata.org/docs/user_guide/dsintro.html#dataframe-interoperability-with-numpy-functions) of pandas documentation to see their naming style.

Reply:
Changed df to ser